modelling_opponent/simulated_mouse.py

```diff
@@ -0,0 +1,104 @@
+import random
```

Comment:
Simulated mouse file is not needed any more.

Reply:
removed